README.md

```diff
@@ -43,7 +43,7 @@ At least three hosts are needed for this example, a primary, a secondary, and te
   - Create a xfs partition and configure it
   - Configure docker
 
-More information about the prerequisites can be found in the following [page](https://www.elastic.co/guide/en/cloud-enterprise/2.7/ece-prereqs.html).
+More information about the prerequisites can be found in the following [page](https://www.elastic.co/guide/en/cloud-enterprise/current/ece-prereqs.html).
```

Comment:
This change is from your other PR :) can you remove it please ?

Reply:
Ah, sorry! Yes. I'm rather new to fumbling around in the GitHub GUI here, I'm more of a git CLI kind of fellow. I'll try to get it sorted out.